package.json

```diff
@@ -4,10 +4,10 @@
   "description": "A set of utilities for building Redux applications in Google Chrome Extensions.",
   "main": "lib/index.js",
   "scripts": {
-    "lint": "./node_modules/.bin/eslint src/**/*.js && ./node_modules/.bin/eslint test/**/*.js",
-    "prepublish": "./node_modules/.bin/babel src --out-dir lib",
-    "pretest": "./node_modules/.bin/babel src --out-dir lib",
-    "test": "npm run-script lint && ./node_modules/.bin/mocha --compilers js:babel-core/register"
+    "lint": "eslint src/**/*.js && eslint test/**/*.js",
```

Comment:
I don't have `eslint` in my path, and I don't think I have it installed in npm global (`npm ls -g | grep eslint` is empty).   

`npm lint` runs for me on windows (as do the other scripts).